org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java

```diff
@@ -904,7 +910,7 @@ public boolean apply(JvmFeature input) {
 	/**
 	 * @since 2.17
 	 */
-	protected void proposeFavoritStaticFeatures(EObject context,ContentAssistContext contentAssistContext,
+	protected void proposeFavoriteStaticFeatures(EObject context,ContentAssistContext contentAssistContext,
```

Comment:
[dietrich@dietrich-thinkpad-x230 xtext-umbrella]$ ./gitAll grep favourite | wc -l
53
[dietrich@dietrich-thinkpad-x230 xtext-umbrella]$ ./gitAll grep favorite | wc -l
49

seems like we cannot decide if we go for BE or AE

Reply:
:(